src/connectivities.jl

```diff
@@ -96,11 +92,11 @@ Materialize a face using the `connec` list and a global vector of `points`.
 """
 function materialize(connec::Connectivity{PL},
                      points::AbstractVector{P}) where {PL<:Polytope,P<:Point}
-  PL(view(points, SVector(connec.indices...)))
+  PL(view(points, connec.indices))
 end
 
 function Base.show(io::IO, c::Connectivity{PL}) where {PL}
   name = prettyname(PL)
   inds = c.indices
-  print(io, "$name$inds")
+  print(io, "$name($(join(inds, ", ")))")
```

Comment:
It would be cleaner to just define inds = Tuple(c.indices) and leave the print line unchanged.